README.md

```diff
@@ -59,39 +61,24 @@ and you should be good to go !
 [2]: https://github.com/fabi1cazenave/kalamine
 [3]: https://github.com/fabi1cazenave/kalamine#installing-distributable-layouts
 [4]: https://github.com/Nuclear-Squid/ergol/blob/master/layouts/ergol.toml
+[5]: https://github.com/pypa/pipx?tab=readme-ov-file#install-pipx
 
 
-Install ErgoL under Arch Linux X.Org
---------------------------------------------------------------------------------
-
-### Create [pyenv](https://wiki.archlinux.org/title/Python/Virtual_environment)
-
-```bash
-python -m venv /path/to/pyenv      # Create a pyenv if you don't already have one
-cd /path/to/pyenv/bin
-sudo su                            
-./python -m pip install kalamine --break-system-packages  # Install kalamine in the actual pyenv
-exit                               # Get out of admin mode
-
-cd ~/.local/bin                    
-ln -s /path/to/pyenv/bin/kalamine  # Create a link to kalamine
-ln -s /path/to/pyenv/bin/xkalamine # Create a link to xkalamine
-```
-
-**At your own risk**: there is also an AUR (Arch User Repository) installer that
-exists: https://aur.archlinux.org/packages/kalamine-git
+### Install (Linux only)
 
-### Install the layout
+`xkalamine` is a linux‑specific tool (shipped with `kalamine`) to easily
+install layouts on Linux. You can drop `sudo` if you use wayland, but it’s
+required on Xorg.
 
 ```bash
-kalamine ergol.toml                # Build the layout
 sudo xkalamine install ergol.toml  # Install the layout
 ```
 
-### Set the layout
+Then switch to ergol using the following command :
 
 ```bash
-setxkbmap fr -variant ergol        # Set the layout
+setxkbmap us  # switch to your usual layout, to go back to it with a simple ctrl-r
+setxkbmap fr -variant ergol  # Switch your keyboard layout to ergol
```

Comment:
That’s what I do as well but I think this is too confusing. Let users enjoy Ergo‑L a bit. :-)

Reply:
You should probably mention that this only works on XOrg.

On both XOrg and Wayland, the system’s keyboard prefs should handle this fine **but** the 1dk layer might not work unless `Ergo‑L` is set as the default keyboard — and there’s not much we can do on our side.

So maybe a line like this could be added:

> if the main dead key or the AltGr layer is not working on your system, setting Ergo‑L as the default layout should solve the issue. We’re sorry about the inconvenience but that’s an XKB thing and there’s nothing we can do on our side.